file-downloader/src/main/java/FileDownloader.java

```diff
@@ -0,0 +1,68 @@
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class FileDownloader {
+
+    public static void main(String[] args) {
+        if (args.length != 2) {
+            System.out.println("Usage: java FileDownloaderWithProgressBar <URL> <outputFile>");
+            System.exit(1);
+        }
+
+        String fileURL = args[0];
+        String outputFile = args[1];
+
+        try {
+            downloadFile(fileURL, outputFile);
+            System.out.println("\nFile downloaded successfully!");
+        } catch (IOException e) {
+            System.err.println("Error: " + e.getMessage());
+        }
+    }
+
+    public static void downloadFile(String fileURL, String outputFile) throws IOException {
+        URL url = new URL(fileURL);
+        HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+
+        int responseCode = connection.getResponseCode();
+        if (responseCode == HttpURLConnection.HTTP_OK) {
+            long contentLength = connection.getContentLengthLong();
+            System.out.println("File size: " + contentLength + " bytes");
```

Comment:
Use logger for all logging

Reply:
Don't forget to implement handling of situation when connection is lost and you have to reconnect and continue with file download.